src/libs/Altinn.Authorization.Integration/src/Altinn.Authorization.Integration.Platform/ResourceRegister/ResourceGetEndpoint.cs

```diff
@@ -0,0 +1,249 @@
+    /// </summary>
+    [JsonPropertyName("visible")]
+    public bool Visible { get; set; }
+
+    /// <summary>
+    /// Gets or sets the competent authority responsible for the resource.
+    /// </summary>
+    [JsonPropertyName("hasCompetentAuthority")]
+    public ResourceHasCompetentAuthority HasCompetentAuthority { get; set; }
+
+    /// <summary>
+    /// Gets or sets a list of keywords associated with the resource.
+    /// </summary>
+    [JsonPropertyName("keywords")]
+    public List<object> Keywords { get; set; }
+
+    /// <summary>
+    /// Gets or sets the access list mode for the resource.
+    /// </summary>
+    [JsonPropertyName("accessListMode")]
+    public string AccessListMode { get; set; }
+
+    /// <summary>
+    /// Gets or sets a value indicating whether self-identified users can access the resource.
+    /// </summary>
+    [JsonPropertyName("selfIdentifiedUserEnabled")]
+    public bool SelfIdentifiedUserEnabled { get; set; }
+
+    /// <summary>
+    /// Gets or sets a value indicating whether enterprise users can access the resource.
+    /// </summary>
+    [JsonPropertyName("enterpriseUserEnabled")]
+    public bool EnterpriseUserEnabled { get; set; }
+
+    /// <summary>
+    /// Gets or sets the type of the resource.
+    /// </summary>
+    [JsonPropertyName("resourceType")]
+    public string ResourceType { get; set; }
+
+    /// <summary>
+    /// Gets or sets a list of authorization references associated with the resource.
+    /// </summary>
+    [JsonPropertyName("authorizationReference")]
+    public List<ResourceAuthorizationReference> AuthorizationReference { get; set; }
+}
+
+/// <summary>
+/// Represents an authorization reference for a resource.
+/// </summary>
+public class ResourceAuthorizationReference
+{
+    [JsonPropertyName("id")]
+    public string Id { get; set; }
+
+    [JsonPropertyName("value")]
+    public string Value { get; set; }
+}
+
+/// <summary>
+/// Represents a description of a resource in multiple languages.
+/// </summary>
+public class ResourceDescription
+{
+    [JsonPropertyName("en")]
+    public string En { get; set; }
+
+    [JsonPropertyName("nb")]
+    public string Nb { get; set; }
+
+    [JsonPropertyName("nn")]
+    public string Nn { get; set; }
+}
+
+/// <summary>
+/// Represents the competent authority responsible for a resource.
+/// </summary>
+public class ResourceHasCompetentAuthority
+{
+    [JsonPropertyName("name")]
+    public CompetentAuthorityName Name { get; set; }
+
+    [JsonPropertyName("organization")]
+    public string Organization { get; set; }
+
+    [JsonPropertyName("orgcode")]
+    public string Orgcode { get; set; }
+
+    /// <summary>
+    /// Represents the name of the competent authority in multiple languages.
+    /// </summary>
+    public class CompetentAuthorityName
+    {
+        [JsonPropertyName("en")]
+        public string En { get; set; }
+
+        [JsonPropertyName("nb-no")]
```

Comment:
nb-no og nn-no er ikke egentlig gyldige språk koder.
Så de ressursene som evt. har brukt disse kan vi egentlig bare ignorere.

Reply:
right, skal fjerne det, så bare at de eksisterte for noen :D